javascript/builtins/RegExp.json

```diff
@@ -723,22 +723,22 @@
                 "version_added": "62"
               },
               "edge": {
-                "version_added": null
+                "version_added": false
               },
               "edge_mobile": {
                 "version_added": null
               },
               "firefox": {
-                "version_added": null
+                "version_added": false
               },
               "firefox_android": {
-                "version_added": null
+                "version_added": false
               },
               "ie": {
                 "version_added": null
               },
               "nodejs": {
-                "version_added": null
+                "version_added": 9
```

Comment:
You'll want this to be a string of the exact node version it was added, so something like "9.10.12" (which is probably not the actual version it was added, but that's what's expected).